src/components/ec-donut/ec-donut.spec.js

```diff
@@ -56,11 +56,11 @@ describe('EcDonut', () => {
 
   it('should get the correct computed properties', () => {
     const wrapper = mountDonut();
-    const circumference = 2 * Math.PI * wrapper.vm.radius;
+    const circumference = 2 * Math.PI * wrapper.vm.RADIUS;
     expect(wrapper.vm.percentageUsed).toBe(20);
     const offset = circumference * (1 - wrapper.vm.percentageUsed / 100);
-    expect(wrapper.vm.dashArray).toBe(circumference);
-    expect(wrapper.vm.dashOffset).toBe(offset);
+    expect(wrapper.vm.circumference).toBe(circumference);
+    expect(wrapper.vm.usedArcLength).toBe(offset);
```

Comment:
A test like these can be removed, we shouldn't test internals of the component like state and computed props. Accessing `wrapper.vm` is a bad thing, it makes tests less maintainable. Checking circumference and usedArcLength should be tested by checking their values in the snapshot and it's already tested on line 29.

More in the docs:

https://github.com/Ebury/eburyonline/blob/master/docs/FRONTEND.md?plain=1#L409

The only acceptable place to use `wrapper.vm` is when testing `v-model`:

https://github.com/Ebury/eburyonline/blob/master/docs/FRONTEND.md?plain=1#L447